lib/ansible/modules/cloud/azure/azure_rm_cdnendpoint_facts.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Hai Cao, <t-haicao@microsoft.com>
```

Comment:
still 2018 and e-mail

Reply:
will fix year. email works here, but not -author field